db/migrations/1649259545966-OrderAndOrderItem.ts

```diff
@@ -0,0 +1,163 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+import { OrderStatus } from '../../src/modules/order/order.enum';
+
+export class OrderAndOrderItem1649259545966 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'order',
+        columns: [
+          {
+            name: 'id',
+            type: 'uuid',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'uuid',
+          },
+          {
+            name: 'status',
+            type: 'varchar',
+            length: '255',
+            isNullable: false,
+            enum: [OrderStatus.PAID, OrderStatus.OPEN, OrderStatus.REOPENED],
+          },
+          {
+            name: 'subTotal',
+            type: 'decimal',
+            isNullable: false,
+          },
+          {
+            name: 'tax',
+            type: 'decimal',
+            isNullable: false,
+          },
+          {
+            name: 'discount',
+            type: 'decimal',
+            isNullable: false,
+          },
+          {
+            name: 'grandTotal',
+            type: 'decimal',
+            isNullable: false,
+          },
+          {
+            name: 'cost',
+            type: 'decimal',
+            isNullable: false,
+          },
+          {
+            name: 'company_id',
+            type: 'uuid',
+            isNullable: false,
+          },
+          {
+            name: 'user_id',
+            type: 'uuid',
+            isNullable: false,
+          },
+          {
+            name: 'created_at',
+            type: 'timestamptz',
+            default: 'CURRENT_TIMESTAMP',
+            isNullable: false,
+          },
+          {
+            name: 'updated_at',
+            type: 'timestamptz',
+            default: 'CURRENT_TIMESTAMP',
+            isNullable: false,
+          },
+        ],
+        foreignKeys: [
+          {
+            name: 'order_company_key',
+            columnNames: ['company_id'],
+            referencedTableName: 'company',
+            referencedColumnNames: ['id'],
+          },
+          {
+            name: 'order_user_key',
+            columnNames: ['user_id'],
+            referencedTableName: 'user',
```

Comment:
Shouldn't an order be related to a Client/Customer instead of a plain User?

Reply:
Well an option is to relate it also with the customer, what do you think @matheo97 ?

Reply:
I think relating this table to Customer would be correct, as User's table is meant to be used for the users of the app (admins, waiter/waitress, etc...)